views/about.erb

```diff
@@ -0,0 +1,60 @@
+<html>
+  <head>
+    <title>"Meighs' Page"</title>
+    <meta name="viewport" content="width=device width">
+  </head>
+  <body>
+      <header><center>
+        <h1>About Me</h1>
+        <p>Here's some stuff you should know about me...</p>
+        <img src = "./image1.jpg" width = "200" height = "150">
+      </center></header><br>
+
+    <nav><center>
+      <a href = "./index.html">Main Page</a>
+      <a href = "./projects.html">My Projects</a>
+      <a href = "./blog.html">My Blog</a>
+      <a href = "./about.html">About Me</a>
+    </center></nav><br>
+
+    <section>
+      <ul style="list-style: none;"><center>
```

Comment:
I would recommend putting the opening and closing `center` tags on a separate line by itself